plugin-development/visual-studio-code-extension.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Amplication Visual Studio Code Extension'
+description: 'Learn how to install and use the Amplication VS Code extension to streamline plugin development.'
+sidebarTitle: 'VS Code Extension'
+icon: 'keyboard'
+---
+
+The Amplication VS Code extension provides a streamlined way to develop and manage plugins for Amplication directly from your IDE. This extension offers features to download plugin templates, manage dependencies, build, push, and publish new plugin versions effortlessly.
+
+## Features
+
+- **Download Plugin Templates**: Select and download pre-existing templates from Amplication’s GitHub repositories.
+- **Manage Dependencies**: Automatically install dependencies for downloaded templates.
+- **Build and Push**: Build your plugin and push updates to your repository.
+- **Version Publishing**: Automate version bumping and publishing directly from VS Code.
+- **Seamless Integration**: Works directly with Amplication’s plugin development workflow.
```

Comment:
unlike the first four items, the last one is not a feature

Reply:
yup